src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -131,7 +131,8 @@ public function load(array $configs, ContainerBuilder $container)
 
         if ($this->isClassificationEnabled($config)) {
             $loader->load('category.xml');
-            $container->setAlias('sonata.media.manager.category', $config['category_manager']);
+            $categoryManager = $container->setAlias('sonata.media.manager.category', $config['category_manager']);
```

Comment:
Looks like `setAlias` does not behave consistently for all Travis jobs

Reply:
Yes, Symfony 2.8 and 3.3 don't return the alias with `setAlias`. Fixing it right now.